opts.go

```diff
@@ -66,3 +75,19 @@ func WithSnapshot(memFilePath, snapshotPath string, opts ...WithSnapshotOpt) Opt
 		m.Handlers.FcInit = loadSnapshotHandlerList
 	}
 }
+
+// WithMemoryBackend sets the memory backend to the given type, using the given
+// backing file path (a regular file for "File" type, or a UFFD socket path for
+// "Uffd" type).
+//
+// Note that if MemFilePath is already configured for the snapshot config, it
+// will be ignored, and the backendPath specified here will be used instead.
+func WithMemoryBackend(backendType, backendPath string) WithSnapshotOpt {
+	return func(cfg *SnapshotConfig) {
+		cfg.MemBackend = &models.MemoryBackend{
+			BackendType: String(backendType),
+			BackendPath: String(backendPath),
+		}
+		cfg.MemFilePath = ""
```

Comment:
nit:
What do you think not clear the `MemFilePath` since it will be ignored by firecracker any way?  
Just set the `MemoryBackend` to align the function description.  
Seems to me clear or not clear both are ok, but users may have to dig deep to find where their previous set value turned empty.

Reply:
Seems OK to me - Done